src/app/dataset/depublication/depublication.component.ts

```diff
@@ -150,10 +152,12 @@ export class DepublicationComponent extends DataPollingComponent {
   */
   checkAllAreSelected(): void {
     if (this.depublicationRows) {
+      this.selectAllDisabled = this.depublicationRows.length === 0;
```

Comment:
Wouldn't it be better to let this depend on the size of enabledRows instead of depublicationRows? Will give us some nicer behavior in case all rows are disabled.